spec/rails_helper.rb

```diff
@@ -22,6 +22,8 @@
 # require features_helper after support files have been loaded
 require "features_helper"
 
+Capybara.server = :puma, { Silent: true }
+
```

Comment:
What's this?

Reply:
This removes the extra output on start of Capybara server in test, which is just ugly and unnecessary